test/util_test.go

```diff
@@ -37,6 +37,7 @@ func TestAssertCanvasTappableAt(t *testing.T) {
 }
 
 func TestAssertImageMatches(t *testing.T) {
+	fyne.CurrentApp().Settings().SetTheme(test.MonoTheme())
```

Comment:
I think this line isn't needed - the theme should come from the current app config